README.dependencies

```diff
@@ -85,8 +85,8 @@ Libraries/Deps
 
   optional
   --------
-  aqbanking		4.0.0			 online banking; also requires
-						 gwehywfar and ktoblzcheck
+  aqbanking		5.3.4			 online banking; also requires
+						 gwehywfar.
```

Comment:
Just to be clear here: does this mean ktoblzcheck is not needed anymore at all ? I understood it only to be dropped for aqbanking 6.

Reply:
Or did you remove it as it's a dependency of aqbanking and not a direct dependency of gnucash ?

Reply:
I suppose it's no longer needed, @cstim removed it entirely in https://github.com/Gnucash/gnucash/commit/dac94ce03806d5791ac7a8a0ffcba03896e8c237#diff-7d8b963127def26b06716af6e17b35d0.

Reply:
Oh, right. I forgot about that bit. So yay, one dependency less!